src/processor/k8sattributesprocessor/options.go

```diff
@@ -37,6 +37,8 @@ const (
 	filterOPDoesNotExist = "does-not-exist"
 	metadataPodStartTime = "k8s.pod.start_time"
 	specPodHostName      = "k8s.pod.hostname"
+	// TODO: use k8s.cluster.uid from semconv when available, and replace clusterUID with conventions.AttributeClusterUid
```

Comment:
Can you create JIRA and mention it here instead of TODO comment which will be forgotten? Thank

Reply:
That's not our comment - it's from the upstream repo.